### PR TITLE
Fix styler excel example - take 2

### PR DESCRIPTION
**Pull request by janosh**
TypeError: <lambda>() takes 0 positional arguments but 1 was given.

Following https://github.com/pandas-dev/pandas/pull/49971#issuecomment-1334262698.

### CODE REVIEW

File: pandas/io/formats/style_render.py [hunk 1]
```diff
@@ -1110,7 +1110,7 @@ def format(
 
         >>> df = pd.DataFrame({"A": [1, 0, -1]})
         >>> pseudo_css = "number-format: 0§[Red](0)§-§@;"
-        >>> df.style.applymap(lambda: pseudo_css).to_excel("formatted_file.xlsx")
+        >>> df.style.applymap(lambda v: pseudo_css).to_excel("formatted_file.xlsx")
         ...  # doctest: +SKIP
```
Comment (phofl) [inline pandas/io/formats/style_render.py:1114]: Can we remove this now?

Comment (janosh) [reply pandas/io/formats/style_render.py:1114]: You mean `# doctest: +SKIP`?

Comment (phofl) [reply pandas/io/formats/style_render.py:1114]: Yep

**Review by phofl — APPROVED** [review]

**Comment by mroeschke** [comment]
Thanks @janosh

**Pull request merged**